### PR TITLE
Combine player prefabs and add Crow spawn/reset to 'boundary' scene

### PR DESCRIPTION
**Pull request by DevonHenderson**
I've worked a bit on getting the new player and crow functioning in the `BoundaryTestScene` as I figure that is likely to be the basis for our main scene.

Current Features:
 - New player prefab used (with PlayerController script added)
 - Player tag added to entire Player prefab for catchup/gameover triggers
 - Added to GameManager to Start and Stop crow spawning
 - Added CrowManager to scene
 -  Removed CrowManager start being used to control initial crow spawning and moved to game manager entirely (was causing issue where the crow was technically being moved twice so would despawn in half the time)

Current issues:
 - When the player enters the 'Catchup' trigger area, the crows speed does not in any way increase to match the faster moving ground. This results in a slightly jittery movement of the crow when it appears to only move a little/almost lock in place. I am unsure how to modify my speed calculations to accurately reflect this potential change in game speed. Might need some help getting this fixed if it is too jarring

### CODE REVIEW

File: Assets/Scripts/Game/GameManager.cs [hunk 1]
```diff
@@ -47,16 +47,19 @@ private void Awake()
...
     private float currentVelocity = 0f;
-  
+
 
     private bool isCatchingUp;
     public bool playerInCatchupZone = false;
 
+    [Header("Crow settings")]
+    [SerializeField] private GameObject crowManagerParent;
+    private CrowManager crowManager;
```
Comment (JohnathanGlasgow) [inline Assets/Scripts/Game/GameManager.cs:62]: Why not expose the CrowManager component itself in the inspector? Then you don't need to get the component from the GameObject on lines 157 and 167

**Review by JohnathanGlasgow — APPROVED** [review]
In CrowManager, you should move `yield return new WaitForSeconds(Random.Range(minSpawnDelay, maxSpawnDelay));` to the beginning of the coroutine. That way the crow doesn't spawn immediately.

In regards to the crow speed issue, this seems to do the job (crowSpeed is declared as a class field, I have it set to 5f):
`
        //Move the crow over time from a start to end point
        while (crow.transform.position.x > endPos.x)
        {
            Vector3 position = crow.transform.position;
            position.x -= Time.deltaTime * (crowSpeed + GameManager.instance.Speed); //Move crow towards end point
            crow.transform.position = position;
            yield return null;
        }
`

**Comment by JohnathanGlasgow** [comment]
Also, try and avoid editing preexisting scenes for feature branches. Safer to just make a copy.

**Review by LiamMcKenzie — APPROVED** [review]
I've noticed no issues with this branch. Should be good to merge into staging

**Pull request merged**